lib/kazoo/cluster.rb

```diff
@@ -33,6 +33,13 @@ def brokers
       end
     end
 
+    def consumergroups
+      @consumergroups ||= begin
+        consumers = zk.get_children(path: "/consumers")
```

Comment:
I checked to be sure.
- Turns out that `rc != 0` if the node doesn't exist.
- If the node exists but doesn't have any children, `result[:children]` will be an empty array.

So this should be good :+1: